packages/edit-site/src/components/editor/index.js

```diff
@@ -263,6 +263,18 @@ export default function EditSiteEditor( { isPostsList = false } ) {
 			) }
 			{ isReady && (
 				<Editor
+					as={ motion.div }
```

Comment:
Thanks for trying things out.

I tried to hack in the browser too but `.edit-site-resizable-frame__inner-content` has overflow hidden so it's very tricky!

I know I mentioned it, but I hacked it by sending a prop to the resizable iframe:

[`119c8e5` (#65619)](https://github.com/WordPress/gutenberg/pull/65619/commits/119c8e5d481cfd8a7f40a50f5d14e79996723bd2#diff-bfbf96cfd600fa9330d907fc11ed2180db094647a9d305099be9ec04cd75b7efR250)

I know that's not ideal as you [mentioned](https://github.com/WordPress/gutenberg/pull/66851#issuecomment-2522110154).